src/apps/api/tests/test_functional.py

```diff
@@ -0,0 +1,58 @@
+from tests.base import SeleniumTestCase
+from producers.models import Producer
+from competitions.models import Competition
+from django.urls import reverse
+from time import sleep
+
+
+class TestChangingProviderAndResultsChange(SeleniumTestCase):
+
+    def test_setup_and_login_as_user(self):
+        #  Starts a new window for Chrome
+        #  Assures they can login
+        self.get(reverse('login'))
+        self.login()
+        self.assertCurrentUrl('/?')
+
+    def test_logging_in_as_producer(self):
+        #  Discovered how to do so on accident, might as well go with it now wheee
+        self.get(reverse('producers:management'))
+        self.login()
+        self.assertCurrentUrl(reverse('producers:management'))
+
+    def test_changing_provider_changes_url(self):
```

Comment:
fix this title to represent the outcomes the test expects (url change AND results change)